### PR TITLE
ECU Reset

### PR DESCRIPTION
**Pull request by DanielConstantinescu**

### CODE REVIEW

File: src/ecu_simulation/BatteryModule/Makefile [hunk 1]
```diff
@@ -4,8 +4,9 @@ CFLAGSTST = -std=c++17 -I/include -DUNIT_TESTING_MODE
 LDFLAGS = -lspdlog
 CFLAGSTST2 = -lgtest -lgtest_main -pthread
 #Sources files
-SRCS = src/main.cpp src/BatteryModule.cpp ../../utils/src/GenerateFrames.cpp src/HandleFrames.cpp ../../utils/src/CreateInterface.cpp src/ReceiveFrames.cpp ../../uds/read_data_by_identifier/src/ReadDataByIdentifier.cpp ../../mcu/src/HandleFrames.cpp ../../mcu/src/ReceiveFrames.cpp ../../mcu/src/MCUModule.cpp
+SRCS = src/main.cpp src/BatteryModule.cpp ../../utils/src/GenerateFrames.cpp src/HandleFrames.cpp ../../utils/src/CreateInterface.cpp src/ReceiveFrames.cpp ../../uds/read_data_by_identifier/src/ReadDataByIdentifier.cpp ../../uds/ecu_reset/src/ecu_reset.cpp ../../mcu/src/HandleFrames.cpp ../../mcu/src/ReceiveFrames.cpp ../../mcu/src/MCUModule.cpp
 SRCSTEST = src/HandleFrames.cpp src/BatteryModule.cpp src/ReceiveFrames.cpp ../../utils/src/GenerateFrames.cpp ../../utils/src/CreateInterface.cpp ../../utils/src/Logger.cpp ../../uds/read_data_by_identifier/src/ReadDataByIdentifier.cpp ../../mcu/src/HandleFrames.cpp ../../mcu/src/ReceiveFrames.cpp ../../mcu/src/MCUModule.cpp
```
Comment (Mihnea9901) [inline src/ecu_simulation/BatteryModule/Makefile:8]: can you please add your file to SRCSTEST too? so tests file can be compiled too

File: src/mcu/include/HandleFrames.h [hunk 2]
```diff
@@ -19,7 +19,10 @@
 #include <vector>
 #include "../include/MCULogger.h"
 #include "../../uds/read_data_by_identifier/include/ReadDataByIdentifier.h"
+#include "../../uds/ecu_reset/include/ecu_reset.h"
 namespace MCU
+
+class HandleFrames
```
Comment (Mihnea9901) [inline src/mcu/include/HandleFrames.h:25]: remove this duplicate

File: src/ecu_simulation/BatteryModule/Makefile (hunk 1 above)
Comment (DanielConstantinescu) [reply src/ecu_simulation/BatteryModule/Makefile]: done

File: src/mcu/include/HandleFrames.h (hunk 2 above)
Comment (DanielConstantinescu) [reply src/mcu/include/HandleFrames.h]: done

**Review by Mihnea9901 — APPROVED** [review]

**Review by TheoRandstad — APPROVED** [review]
Looks good.

**Review by RubenNatanael — APPROVED** [review]

**Pull request merged**